src/config/sites_fixes_v2.json

```diff
@@ -247,6 +247,14 @@
             "rules": [
                 "#watch-appbar-playlist * { -webkit-filter: none !important; }"
             ]
+        },
+        {
+            "url": [
```

Comment:
This array can be replaced with "music.yandex.*". Also please keep an alphabetical order by domain name.